topology/Kansas University/KANSAS_HOSTED/KANSAS_HOSTED_BOSCO_CE.yaml

```
@@ -23,7 +23,5 @@ Resources:
       CE:
         Description: Compute Element
         Details:
-          hidden: false
-    VOOwnership:
-      OSG: 100
+          hidden: falses
```

@yzz971115 this looks like this change was made by mistake

```suggestion
          hidden: false
```